src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -151,6 +190,12 @@ export default {
 		this.lenderInfo = cachedLenderInfo?.community?.lender ?? {};
 		this.allAchievements = cachedLenderInfo?.userAchievementProgress?.achievementProgress ?? [];
 		this.lenderIsPublic = !!this.lenderInfo?.id;
+
+		const utmCampaign = this.$route?.query?.utm_campaign ?? '';
+		if (utmCampaign.includes('badge_') && utmCampaign.includes('social_share')) {
+			const badgeKey = utmCampaign.split('badge_')[1];
+			this.loadBadgeInfo(badgeKey);
```

Comment:
Do we need `await` this query so that the data is guaranteed to be there on server render? I see that we're pulling data from this query to add to the page title and whatnot.